app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [OPTIONAL] This method here for updating the `update_comments_counter` is correct and should work as expected, but using `increment!` is typically more efficient and preferable for this specific task.  The solution using `post.increment!(:comments_counter)` is generally preferred for updating a counter attribute like comments_counter because it's more efficient, handles concurrency well, and is more readable. I'll suggest you take this into account and update your code accordingly. :+1: :100: